releases/latest/snaps.yaml

```diff
@@ -0,0 +1,16 @@
+packages:
+- name: grafana-agent
+  push_channel: latest/stable
+  revision: 51
+- name: grafana-agent
+  push_channel: latest/stable
+  revision: 82
```

Comment:
It's expected to have two revisions of the same snap for `grafana-agent`: one is used for `strict` mode and the other for `classic` mode